python/hail/methods/family_methods.py

```diff
@@ -225,4 +216,207 @@ def mendel_errors(dataset, pedigree):
 
     kts = require_biallelic(dataset, 'mendel_errors')._jvds.mendelErrors(pedigree._jrep)
     return Table(kts._1()), Table(kts._2()), \
-           Table(kts._3()), Table(kts._4())
\ No newline at end of file
+           Table(kts._3()), Table(kts._4())
+
+@handle_py4j
+@typecheck(dataset=MatrixTable,
+           pedigree=Pedigree)
+def tdt(dataset, pedigree):
+    """Performs the transmission disequilibrium test on trios.
+
+    .. include:: ../_templates/req_tvariant.rst
+
+    .. include:: ../_templates/req_biallelic.rst
+
+    Examples
+    --------
+    Compute TDT association results and export to a file:
+
+    .. testsetup::
+    
+        tdt_dataset = hc.import_vcf('data/tdt_tiny.vcf')
+
+    >>> pedigree = Pedigree.read('data/tdt_trios.fam')
+    >>> tdt_table = methods.tdt(tdt_dataset, pedigree)
+    >>> tdt_table.export('output/tdt_results.tsv')
+
+    Export only variants with p-values below 0.001:
+
+    >>> tdt_table = tdt_table.filter(tdt_table.pval < 0.001)
+    >>> tdt_table.export("output/tdt_results.tsv")
+
+    Notes
+    -----
+    The
+    `transmission disequilibrium test <https://en.wikipedia.org/wiki/Transmission_disequilibrium_test#The_case_of_trios:_one_affected_child_per_family>`__
+    tracks the number of times the alternate allele is transmitted (t) or not
+    transmitted (u) from a heterozgyous parent to an affected child under the
```

Comment:
unclear wording. It tracks the number of times the alternate allele is transmitted and compares to the null expectation of 0.5 transmissions per allele.